DOCSUP-11549: Document the compile_aggregate_expressions and min_count_to_compile_aggregate_expression settings (#30860)
ClickHouse/ClickHouse: docs/en/operations/settings/settings.md

Default value: `3`.

## compile_aggregate_expressions {#compile_aggregate_expressions}

> Enables or disables JIT-compilation of aggregate functions to native code. Enabling the `compile_aggregate_expressions` setting can improve the performance.

Reviewer: ```suggestion
Enables or disables JIT-compilation of aggregate functions to native code. Enabling this setting can improve the performance.
```

Possible values:

-   0 — Aggregation is done without JIT compilation.
-   1 — Aggregation is done using JIT compilation.

Default value: `0`.

**See Also** 

-   [min_count_to_compile_aggregate_expression](#min_count_to_compile_aggregate_expression)

## min_count_to_compile_aggregate_expression {#min_count_to_compile_aggregate_expression}

> If the [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled and there are more then `min_count_to_compile_aggregate_expression` identical aggregate expressions in a `SELECT` query, then they are compiled JIT.

Reviewer: ```suggestion
If the [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled and there are more than `min_count_to_compile_aggregate_expression` identical aggregate expressions in a `SELECT` query, then they are JIT-compiled.
```

Reviewer: Maybe it would be better to say

> The minimum number of identical aggregate expressions in a `SELECT` query before they are JIT-compiled. Works only if [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled.

> If the [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled and there are more then `min_count_to_compile_aggregate_expression` identical aggregate expressions in a `SELECT` query, then they are compiled JIT.

Author: ```suggestion
The minimum number of identical aggregate expressions in a `SELECT` query before they are JIT-compiled. Works only if [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled.
```
